app/controllers/AMIable.scala

```diff
@@ -19,14 +20,23 @@ class AMIable @Inject()(override val amiableConfigProvider: AmiableConfigProvide
 
   def index = AuthAction.async { implicit request =>
     val ssa = SSA(stage = Some("PROD"))
+    val charts = List(
```

Comment:
I think this is too much logic (and hard-coded data) for the edge of the program. An example of why that's a good habit to try and stick to is that you can't currently test the construction of this list. I feel like a function that returns the charts when given the current values out the agents would be a nicer fit for here.

I could go either way on whether this actually needs any tests (possibly the map bit I guess), up to you.